article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -120,14 +119,19 @@ object ArticlePicker {
   }
 
   def getTier(page: PageWithStoryPackage)(implicit request: RequestHeader): RenderType = {
+
     val whitelistFeatures = featureWhitelist(page, request)
     val isEnabled = conf.switches.Switches.DotcomRendering.isSwitchedOn
-    val isBetaUser = ActiveExperiments.isParticipating(DotcomRenderingBeta)
+    val isAdFree = ArticlePageChecks.isAdFree(page, request)
+    val isCommercialBetaUser = ActiveExperiments.isParticipating(DotcomRenderingAdvertisements)
```

Comment:
I wonder whether we can rename this? Or perhaps just start using this term more in the team. As I wasn't initially sure what 'commercialBetaUser' meant. Perhaps something closer to `isInDCRAdTest` ? although that also seems wordy!